src/steps/05-PerformanceRequirements/DOW/OtherOfferingSummary.vue

```diff
@@ -6,18 +6,23 @@
           <h1 class="page-header mb-3">
             Your {{ serviceGroupVerbiageInfo.headingSummary }}
           </h1>
-          <p>
-            If you need more {{ serviceGroupVerbiageInfo.typeForText }}s, add them below. 
-            You can also edit or delete any info from the 
-            {{ serviceGroupVerbiageInfo.typeForText }}s that 
-            you have already entered. When you’re done, click "Continue" and we will 
-            move on to your 
+          <p v-if="!hasSecretOrTopSecretInstance">
+            If you have more requirements for this category, add them below. You can
+            also edit or delete any info from the {{ serviceGroupVerbiageInfo.typeForText }}
+            that you have already entered. When you’re done, click “Continue” and we will
             <span v-if="nextOfferingGroupStr && !returnToDOWSummary">
-              {{ nextOfferingGroupStr }} requirements.
-            </span> 
-            <span v-else>performance requirements summary.</span>
+              move on to your {{ nextOfferingGroupStr }} requirements.
+            </span>
+            <span v-else>wrap up this category.</span>
           </p>
 
+          <p v-if="hasSecretOrTopSecretInstance">
+            If you have more requirements for this category, add them below. You can also
+            edit or delete any info from the {{ serviceGroupVerbiageInfo.typeForText }}
+            that you have already entered. When you’re done, click "Continue" and we will
+            find out about your security requirements for these
```

Comment:
the last part of the line is the only difference between paragraphs. In the interest of keeping it DRY, pls add a span with `v-if="showSecurityNote"` above line 13 with content `find out about your security requirements for these services.` ... then no need for two paragraphs with conditional